mkdocs.yml

```diff
@@ -11,27 +11,30 @@ edit_uri: ""
 nav:
   - Overview: README.md
   - Creating a Cluster:
-      - Multi-Node Cluster:      create-cluster.md
+      - Running k0s as a service: install.md
+      - Multi-Node Cluster:      k0s-multi-node.md
       - Single-Node Cluster:     k0s-single-node.md
       - Docker:                  k0s-in-docker.md
       - Windows (experimental):  experimental-windows.md
+      - Worker node configuration: worker-node-config.md
       - System Requirements:     system-requirements.md
       - Worker node configuration: worker-node-config.md
+      - Install with k0sctl:     k0sctl-install.md 
   - Architecture:
       - Architecture: architecture.md
```

Comment:
We have two menu items now:
Architecture, and
Architecture/Architecture.

Architecture/Architecture should be renamed to `Architecture/Overview`